sklearn/metrics/tests/test_classification.py

```diff
@@ -684,17 +682,11 @@ def test_matthews_corrcoef():
     assert_almost_equal(matthews_corrcoef(y_true, y_true_inv2), -1)
 
     # For the zero vector case, the corrcoef cannot be calculated and should
-    # result in a RuntimeWarning
-    mcc = assert_warns_div0(matthews_corrcoef, [0, 0, 0, 0], [0, 0, 0, 0])
```

Comment:
Since you remove the 2 only uses of assert_warns_div0, I think it's safe to delete it from the _testing module.

Reply:
do you mind completely removing ``assert_warns_div0`` from the code base here https://github.com/scikit-learn/scikit-learn/blob/7b1c9afcaf34e622de76d1f5d5e929e5aaffc514/sklearn/utils/_testing.py#L190 ?

Reply:
Now it is removed.